ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.14.yaml

```diff
@@ -152,6 +152,45 @@ tests:
     env:
       OPENSTACK_AUTHENTICATION_METHOD: password
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-masterscale
+  minimum_interval: 189h
+  steps:
+    cluster_profile: openstack-vexxhost
+    test:
+    - as: openstack-test-cpms
+      commands: |
+        dir=$GOPATH/src/github.com/openshift/cluster-control-plane-machine-set-operator
+        git clone https://github.com/openshift/cluster-control-plane-machine-set-operator --branch main --single-branch $dir
+        export OPENSTACK_CONTROLPLANE_FLAVOR_ALTERNATE="$(<"${SHARED_DIR}/OPENSTACK_CONTROLPLANE_FLAVOR_ALTERNATE")"
+        cd $dir
+        make e2e-presubmit
+      from: tests
```

Comment:
this image doesn't work, I need to find another one, got:

```
/bin/bash: line 2: GOPATH: unbound variable
```

Reply:
for the record, i tried `src` but it didn't work because I need some knobs in this file to build it.